packages/spaces/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import Spaces, { useSpace, SpacesContext } from './types/Spaces';
+import Spaces, { useSpaces, SpacesContext } from './types/Spaces';
```

Comment:
We need to create a quick def for `useSpacesContext`

Reply:
I created it just forgot to export it 🤦‍♂. good catch: 316c69f